std/digest/digest.d

```diff
@@ -1021,3 +1021,53 @@ if (isDigest!T) : Digest
     ubyte[5] buf;
     assert(hash.peek(buf).toHexString() == "39A34F41");
 }
+
+/**
+ * Securely compares two digest representations. This function is $(BIGOH n)
+ * for all strings that are of equal length. Use of this function is recommended
+ * over `==` for comparing verification tokens, as it protects from brute force
+ * timing attacks that can reveal the key.
+ *
+ * Params:
+ *     s1 = A digest representation
+ *     s2 = A digest representation
+ * Returns:
+ *     `true` if both representations are equal, `false` otherwise
+ * See_Also:
+ *     $(LINK2 https://security.stackexchange.com/a/74552, An in depth explanation of
+ *     the attack).
+ */
+bool secureCompare(T)(T[] s1, T[] s2) if (isIntegral!T || isSomeChar!T)
```

Comment:
"compare" in D nomenclature is closer to "cmp" i.e. ordering, so the better name here is `secureEqual`